niworkflows/viz/plots.py

```diff
@@ -37,13 +37,13 @@ def __init__(self, func_file, mask_file=None, data=None, conf_file=None, seg_fil
         func_nii = nb.load(func_file)
         self.tr = tr if tr is not None else func_nii.header.get_zooms()[-1]
 
-        self.mask_data = np.ones_like(func_nii.get_data(), dtype='uint8')
+        self.mask_data = np.ones_like(func_nii.get_fdata(), dtype='uint8')
```

Comment:
This seems like a huge waste. Presumably `func_nii` is 4D and the mask should be 3D. Also, you're creating it and potentially dropping it in the next line. Try:

```suggestion
        self.mask_data = nb.fileslice.strided_scalar(func_nii.shape[:3], np.uint8(1))
```

This creates an array that has one value in it, but appears to have whatever shape you specify. See https://nipy.org/nibabel/reference/nibabel.fileslice.html#strided-scalar